ferrocene/tools/coverage-of-subset/src/main.rs

```diff
@@ -40,12 +56,21 @@ fn handle_dir(dir_path: impl AsRef<Path>) {
 
 fn handle_file(file_path: &Path) {
     let mut file_content = fs::read_to_string(&file_path).unwrap();
-    file_content = inject_coverage_attribute(&file_content);
+    file_content = inject_coverage_attribute(file_content);
     fs::write(file_path, file_content).unwrap();
 }
 
-fn inject_coverage_attribute(file_content: &str) -> String {
-    file_content.replace(MARKER, &MARKER_PLUS_TO_BE_INSERTED)
+fn inject_coverage_attribute(mut file_content: String) -> String {
+    // This is, frankly, a very slow hack with a heavy hand, but it works for our purposes.
+    for applicable in APPLICABLE {
+        for i in 0..MAX_INDENTATION {
+            let valid_marker = format!("{MARKER}\n{}{}", " ".repeat(i), applicable);
```

Comment:
Can we go in steps of four? The formatting should enforce that indentation is a multiple of four, no?

```suggestion
            let valid_marker = format!("{MARKER}\n{}{}", " ".repeat(i * 4), applicable);
```

If so we can probably reduce the size of `MAX_INDENTATION`.

Reply:
This code takes all of like 2s to run on my machine I'm not particularly worried about perf. I'd rather just leave this as is if that's ok.